From: Robert Schwebel <r.schwebel@pengutronix.de>
Date: Thu, 29 Jun 2017 23:03:06 +0200
Subject: [PATCH] init.d/Makefile.am: do not call systemd during compilation

Remove systemd call: we can not assume that the user is allowed to do
so: he might not if he compiles the software as non-root.

When cross-compiling, the intended systemd might not even run on the
same hardware.

We better leave the decision whether or not to call systemd to the
administrator.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>
---
 init.d/Makefile.am | 2 --
 1 file changed, 2 deletions(-)

diff --git a/init.d/Makefile.am b/init.d/Makefile.am
index 4a6e551f0b51..efb01e3c8aab 100644
--- a/init.d/Makefile.am
+++ b/init.d/Makefile.am
@@ -39,6 +39,4 @@ if ENABLE_SYSTEMD_LOOKUP
 else
 	install -p -D -m644 haveged.service $(DESTDIR)$(unit_dir)/haveged.service;
 endif
-	systemctl enable haveged.service;
-
 endif
